prov/efa/src/rxr/rxr_ep.c

```diff
@@ -475,6 +475,9 @@ void rxr_release_tx_entry(struct rxr_ep *ep, struct rxr_tx_entry *tx_entry)
 	if (tx_entry->rxr_flags & RXR_TX_ENTRY_QUEUED_RNR)
 		dlist_remove(&tx_entry->queued_rnr_entry);
 
+	if (tx_entry->state == RXR_TX_QUEUED_CTRL)
```

Comment:
Okay I see you put this in a separate patch. There's no point having a bugfix patch for a bug introduced in the same PR, just squash into original commit.

Reply:
Done in new revision